packages/ui-extensions/src/components/index.ts

```diff
@@ -1,14 +0,0 @@
-export {BlockStack} from './BlockStack/BlockStack';
-export type {BlockStackProps} from './BlockStack/BlockStack';
-
-export {Button} from './Button/Button';
-export type {ButtonProps} from './Button/Button';
-
-export {Text} from './Text/Text';
-export type {TextProps} from './Text/Text';
-
-export {TextField} from './TextField/TextField';
-export type {TextFieldProps} from './TextField/TextField';
-
-export {View} from './View/View';
-export type {ViewProps} from './View/View';
```

Comment:
There was a duplicate "index" file in `src/components.ts`. I switched at some point to avoid the use of `index` files as a bit of an exploration into how that feels, since I suspect index files will be less popular if the convention becomes to always include file extensions in import specifiers. Still not sure how I feel about it, but this file was not being used because of the "top level" one.